cmd/chart/verify.go

```diff
@@ -36,6 +36,7 @@ autoscaler          ghcr.io/openfaasltd/autoscaler:0.2.5
 
 	command.Flags().StringP("file", "f", "", "Path to values.yaml file")
 	command.Flags().BoolP("verbose", "v", false, "Verbose output")
+	command.Flags().IntP("max", "m", 3, "Upto max level")
```

Comment:
`Specify how many levels deep into the YAML structure to walk looking for image: tags.`

Reply:
Done